Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fluent/fluentd-kubernetes-daemonset:v1.2-debian-logzio
+FROM fluent/fluentd-kubernetes-daemonset:v1.3-debian-logzio-1
```

Comment:
I think the latest is `fluent/fluent-kubernetes-daemonset:v1.3.3-debian-logzio-1.1`